lib/utils/storage/secure_storage.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:injectable/injectable.dart';
+
+const String accessTokenKey = "accessTokenKey";
+const String refreshTokenKey = "refreshTokenKey";
+
+AndroidOptions _getAndroidOptions() => const AndroidOptions(
+      encryptedSharedPreferences: true,
+    );
+
+@Injectable()
+class SecureStorage {
+  final _secureStorage = const FlutterSecureStorage();
```

Comment:
`FlutterSecureStorage` should be provided by DI, instead of creating a new const value here